client/src/components/utils/History.js

```diff
@@ -0,0 +1,7 @@
+/*
+* This is a helper to give components access to the "history" object
+*/
+import createBrowserHistory from 'history/createBrowserHistory';
+
+const history = createBrowserHistory();
+export default history;
```

Comment:
This is our helper JS file that basically makes a History object for our application.

I know we have access to the History with "withRouter", but I'm already using Routes in App.js so I can't have withRouter. I do not know how to access history w/o withRouter except for making an entirely new History class.